### PR TITLE
Code Cleanup & Made code ShellCheck compliant

### LINKED ISSUE #20: Clean up code.
- [ ]  Think about the filenaming structure and improve it.
    - [ ] Some files are called `custom_install..` even though they do not install anything.
- [ ] Make code base `shellcheck` compliant.

### PR DESCRIPTION
**Pull request by mohitsaxenaknoldus**
Fixes #20 

I've renamed two files that were not installing anything but were still names with the word install.

The `custom_install...` files mentioned in the issue are named that way because they are part of the custom install process and they looked more intuitive this way to me. However, if you still want me to rename them then let me know I'll do it :)

### CODE REVIEW

File: src/helper.sh [hunk 1]
```diff
@@ -79,16 +87,16 @@ write_to_log() {
...
-	
+
 	shift # eat first argument
 	shift # eat second argument
-	
+
 	# create empty file (overwrite previous content if it exists)
-	> $dest_path
-	
+	: > "$dest_path"
```
Comment (a-t-0) [inline src/helper.sh:95]: I was curious what the : does. Would you care to explain?
(You can ignore this comment)

File: src/helper.sh [hunk 2]
```diff
@@ -79,16 +87,16 @@ write_to_log() {
...
+
 	# append remaining args to file
-	for i in "$*"; do
-		cat <<< "$i" >> $dest_path
+	for i in "$@*"; do
```
Comment (a-t-0) [inline src/helper.sh:98]: I was curious what the @ does. Would you care to explain?
(You can ignore this comment)

**Review by a-t-0 — APPROVED** [review]

**Comment by a-t-0** [comment]
That is a large amount of work! Thank you for going through the code so consistently!

> I've renamed two files that were not installing anything but were still names with the word install. 
Excellent!

> The custom_install... files mentioned in the issue are named that way because they are part of the custom install process and they looked more intuitive this way to me.
Thank you for your initiative, making a pragmatic decision and being transparent about it, I think you solved it well like this.

> However, if you still want me to rename them then let me know I'll do it :)
That will not be necessary, I merged your work into the main branch.

Thank you! I got the GitLab server installation working, so I will spin up the server and see if/how I can set up some mirrors to get that CI badge for these repositories now!

File: src/helper.sh (hunk 1 above)
Comment (mohitsaxenaknoldus) [reply src/helper.sh:95]: not much difference between the two except `: >` is more portable, ShellCheck prefers it over `>`, and it looks like a smiley face :)

**Pull request merged**